articles/introduction-to-elastic-stack/index.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: engineering-education
+status: publish
+published: 
+url: /engineering-education/introduction-to-elastic-stack/
+title: Introduction to Elastic Stack(ELK)
+description: This article will cover the basics of Elastic Stack - a log management platform
+author: elly-omondi
+date: 2021-11-08T00:00:00-01:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+
+   - url: /engineering-education/introduction-to-elastic-stack/hero.jpg
+    alt: ELK cover image 
+---
+
+
+[**Elastic stack** (ELK)](https://www.elastic.co/elastic-stack)  is a bundle of software components that can search, visualize, and manage log files(end to end log analysis).
+<!--more-->
+
+Computing devices perform activities and processes that upon completion result into logs that are stored as files. These log data can be  present in servers(i.e. from completed user authentication processes) as well as mobile devices (i.e from activity log)
+The Elastic stack(ELK) provides a log management platform of these log files
+
+
+
+### Overview
+This article will cover:
+- [What is Elastic Stack](#What-is-elastic-stack)
+- [Components of the Elastic Stack](#Components-of-the-elastic-stack)
+- [How ELK Works](#CHow-ELK-works)
+- [Benefits of using ELK](#Benefits-of-using-ELK)
+
+### What is Elastic Stack
+
+Elastic stack, popularly known as the ELK, is a bundle of software components(tools) that provide the ability to search, view and manage log files. The ELK does the processing of the log files (data in any format) from all systems and applications in use then stores them in a JSON formatted document.
+The ELK gives users the ability to analyze these  logs to create meaningful visualisations, perform infrastructure and system monitoring, do efficient troubleshooting and even identify and raise security issues.
+
+
+The software  suite is widely used because it is open-sourced (both for personal and entreprise use) and it can be scaled easily based on users needs
```

Comment:
```suggestion
The ELK stack is popular because it is an open-source software. Along with this, it is an easily scalable software which makes it the perfect choice for personal use as well as deploying it  in your organization. 

There is also a vendor supported paid version of the software suite with added functionality and added cloud support. 
```